Apps/WebClient/src/ClientApp/app/views/landing.vue

```diff
@@ -468,7 +468,7 @@ import {
   faSyringe,
   IconDefinition,
   faClipboard,
-  faExclamationTriangle,
+  faExclamationTriangle

```

Comment:
Were the commas removed from linting?

Reply:
correct. Not sure whats up with sams linter